docs/faq.rst

```diff
@@ -20,3 +20,19 @@ A: No. ``@most/core`` is focused on reactive event programming rather than Array
 **Q: I want to stream chunked data with flow control (also known as "back pressure"). Should I use ``@most/core`` for that?**
 
 A: No. ``@most/core`` is focused on reactive event programming rather than chunked/block data streaming :ref:`Read more here <when should you use it>`.
+
+**Q: Are ``@most/core`` event streams the same as Node streams?**
+
+A: No. ``@most/core`` and Node streams are fairly different things, each of which happened to choose the name "stream" for one of its core concepts. They have a different primary goals, and thus different architectures and APIs.
+
+``@most/core`` :ref:`Event Streams`:
+
+* Deal with discrete events as as they happen, such as mouse clicks, where reactivity (timeliness) is a major factor
+* Provide an API of functions for filtering, transforming, merging, etc. discrete event streams
+* Add new functionality by writing new event sources and functions
+
+In contrast, Node streams:
+
+* Deal primarily with chunked data IO (even though they have "object mode"), where flow control (aka "back pressure") is a major factor
```

Comment:
```
 In contrast, Node streams

* deal primarily with chunked data IO (even though they have "object mode"), where flow control (a.k.a. "back pressure") is a significant factor;
 * provide an API based around piping one stream through another; and
 * add new functionality by writing new Readable, Writable, and Transform streams.
```

Applying this replaces the overused word _major_ with _significant_, corrects the _a.k.a._ abbreviation to have periods, and sets the vertical list that complete a sentence as per the Chicago Manual of Style.

Reply:
Thanks! Done in e1ff33b.